airgun/browser.py

```diff
@@ -342,7 +342,11 @@ def get_file_content(self, uri):
         )
 
         # it must be local absolute path, without protocol
-        elem.send_keys(unquote(uri[7:]))
+        # In some version <= 98, this changed so schema is not included in the path
+        if 'file://' in uri or 'http://' in uri:
```

Comment:
You're re-using the original string slice here, so I'm going to approve it, but this has always been a magic number that would be easy to 'break'.

I'd suggest doing something like this instead, which uses urllib.parse functions to explicitly deconstruct and reconstruct the URL by its components.

```
from urllib import urlsplit, urlunsplit
urlunsplit(('', *urlsplit(uri)[1:])).lstrip('/')
```

Reply:
Yes, I agree, I've been thinking about that, too. I just went with minimal changes and minimal dependency introduction.